scripts/docker-entrypoint.sh

```diff
@@ -64,10 +64,16 @@ function create_config {
      echo "Found custom configuration, securing and executing it."
      chown root:root /etc/openxpki/customconfig.sh
      chmod 700 /etc/openxpki/customconfig.sh
-     /etc/openxpki/customconfig.sh
+     sh /etc/openxpki/customconfig.sh
+  elif [ -f "/usr/share/doc/libopenxpki-perl/examples/sampleconfig.sh.gz" ]; then
+    echo "Found no custom customconfig.sh - using default compressed sampleconfig.sh.gz from /usr/share/doc/libopenxpki-perl/examples/sampleconfig.sh"
+	gunzip /usr/share/doc/libopenxpki-perl/examples/sampleconfig.sh.gz
```

Comment:
I had to change this line to
```sh
gunzip < /usr/share/doc/libopenxpki-perl/examples/sampleconfig.sh.gz > /usr/share/doc/libopenxpki-perl/examples/sampleconfig.sh
```
Otherwise gunzip complained about hard links on this file (at least with overlay storage driver).